packages/aws-cdk-lib/aws-apigateway/lib/stage.ts

```diff
@@ -471,7 +472,7 @@ export class Stage extends StageBase {
         if (self.enableCacheCluster === undefined) {
           self.enableCacheCluster = true;
         } else if (self.enableCacheCluster === false) {
-          throw new Error(`Cannot enable caching for method ${path} since cache cluster is disabled on stage`);
+          throw new UnscopedValidationError(`Cannot enable caching for method ${path} since cache cluster is disabled on stage`);
```

Comment:
we can probably add scope to this method also. are you trying to do this before defaulting to `Unscoped`?